backend/push.ts

```
@@ -0,0 +1,53 @@
+        if (err) {
+          return reject(err)
+        }
+        if (Array.isArray(sessions)) {
+          resolve(sessions)
+        }
+      })
+    })
+    // Filtere die Sessions nach der Benutzer-ID
+    const userSessions = sessions.filter((session) => {
+      return (
+        session.passport && session.passport.user && session.passport.user._id && session.passport.user._id.toString() === userId.toString()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)